pkg/operator/ceph/file/filesystem.go

```diff
@@ -160,13 +156,31 @@ func newFS(name string, metadataPool *model.Pool, dataPools []*model.Pool, activ
 	}
 }
 
+// SetPoolSize method sets the sizes for MetadataPool and dataPool
+func SetPoolSize(context *clusterd.Context, clusterName, fsName string, dataPool []*model.Pool, metadataPool *model.Pool) error {
+	// generating the metadata pool's name
+	metadataPool = generateMetadataPoolName(fsName, metadataPool)
+	err := client.CreatePoolWithProfile(context, clusterName, *metadataPool, appName)
+	if err != nil {
+		errors.Wrapf(err, "failed to update metadata pool %q", *&metadataPool.Name)
+	}
+	// generating the data pool's name
+	dataPool = generateDataPoolNames(fsName, dataPool)
+	for _, pool := range dataPool {
+		err := client.CreatePoolWithProfile(context, clusterName, *pool, appName)
+		if err != nil {
+			errors.Wrapf(err, "failed to update metadata pool %q", *&pool.Name)
```

Comment:
Return the error here as well

Reply:
This error should indicate the "data" pool, not the metadata pool

Reply:
Thanks, I missed it.